core/frontend/src/test/render/webgl/Decorations.test.ts

```diff
@@ -61,54 +62,54 @@ describe("Decorations", () => {
     TestDecorator.dropAll();
   });
 
-  after(async () => {
+  afterAll(async () => {
     await imodel.close();
     await IModelApp.shutdown();
     document.body.removeChild(div);
   });
 
-  it("draws box decoration in expected location", () => {
+  it("draws box decoration in expected location", { timeout: 20000 }, () => {  // macOS is slow.
```

Comment:
wanted one run in CI before replying - it's not needed anymore. I'm removing all the places where we set the timeouts as high or around 20 seconds. Vitest, by default, has these set to [15 seconds](https://vitest.dev/config/#testtimeout)(because we are using a real browser)